.github/ISSUE_TEMPLATE/lsp_bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Language server client bug report
+about: Report a built-in lsp problem in Nvim
+title: ''
+labels:
+  - bug
+  - lsp
+---
+
+<!-- Before reporting: search existing issues and check the FAQ. -->
+
+- `nvim --version`:
+- Operating system/version:
+- Terminal name/version:
+- language server name:
+
+<details>
+<summary>`nvim -c ":checkhealth"`</summary>
+
+<!-- Paste the results from `nvim -c ":checkhealth"` here. -->
```

Comment:
I would recommand `checkhealth nvim_lsp` since hte plugin is lekely installed but we may need a builtin lsp checkhealth.